test/src/unit-capi-array.cc

```diff
@@ -1118,15 +991,8 @@ TEST_CASE_METHOD(
     ArrayFx,
     "C API: Test opening array at timestamp, writes",
     "[capi][array][open-at][writes]") {
-  std::string temp_dir;
-  if (supports_s3_)
-    temp_dir = S3_TEMP_DIR;
-  else if (supports_azure_)
-    temp_dir = AZURE_TEMP_DIR;
-  else if (supports_hdfs_)
-    temp_dir = HDFS_TEMP_DIR;
-  else
-    temp_dir = FILE_URI_PREFIX + FILE_TEMP_DIR;
+  // TODO: refactor for each supported FS.
```

Comment:
Please create a clubhouse ticket to track this so we don't forget to revisit this later.